packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -221,6 +222,8 @@ class Auth {
             let authorizationMethod = '_loginAsGuest'
             if (credentials) {
                 authorizationMethod = '_loginWithCredentials'
+            } else if (isTokenValid(this.authToken) && this._customerId) {
```

Comment:
Currently using the extra this._customerId to workaround some test issues.

Without this, several unrelated tests (ie. checkout, password reset) that mock isTokenValid to true to get around the login function in _app and avoid mocking Auth end up hitting this still via an unmocked useCustomer.login. 

A more proper workaround is probably to fix those tests mocking.

Reply:
Update: This workaround breaks the actual fix. Going to remove it and try something else